src/UI/Implementation/Component/Input/Field/Renderer.php

```diff
@@ -810,9 +801,6 @@ protected function renderHiddenField(F\Hidden $input): string
     public function registerResources(ResourceRegistry $registry): void
     {
         parent::registerResources($registry);
-        $registry->register('./node_modules/moment/min/moment-with-locales.min.js');
-        $registry->register('./node_modules/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js');
```

Comment:
I am not 100% sure about these two lines. But since we use HTML5 now, we actually do not need Javascript anymore, right? When I tested the Inputs locally without these two lines, everything was still working as intended.